src/index.html

```diff
@@ -6,13 +6,63 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__section card__section--img"></div>
```

Comment:
@mrkovaliv so it's better to delete this unnecessary code block, right?

cards.scss line 13-15th:
`  &__section {
    display: flex;
    align-items: center;
`

this part is quite confusing for me. I mean about parent-child relations.
 
another approach I thought about in case I need general styles for all blocks was next:

`
 <div
      class="card"
    >
      <div class="card__section price">
        <span class="price__label">Price:</span>
        <span class="price__value">$2,199</span>
      </div>
`
will it be correct according to BEM?

Reply:
It's okay if it confuses you at the beginning. With a little more practice, you'll understand everything very well. 👍 

Correct example:

```
    <div class="card__price">
      <span class="card__price-label">Price:</span>
      <span class="card__price-value">$2,199</span>
    </div>
```
`.card `- block
`.card__price` - element of block .card
`.card__price-label` - element of block .card__price
`.card__price-value` - another element of block .card__price
    
 Also you can review this video for better understanding:
 https://www.youtube.com/watch?v=HihYQVuH64U